pkg/gofr/gofr.go

```diff
@@ -34,6 +34,9 @@ import (
 const (
 	defaultPublicStaticDir = "static"
 	gofrTraceExporter      = "gofr"
+	zipkinTraceExporter    = "zipkin"
+	jaegerTraceExporter    = "jaeger"
+	otlpTraceExporter      = "otlp"
```

Comment:
```suggestion
 	traceExporterGoFr      = "gofr"
	traceExporterZipkin    = "zipkin"
	traceExporterJaeger    = "jaeger"
	traceExporterOTLP      = "otlp"
```

Usually, constant are sharing a common prefix. It helps for IDE completion